Editor/PackageManager/PrefabsValidator.cs

```diff
@@ -23,7 +23,7 @@ private static void OnPostProcess(object sender, DependencyManager.DependenciesE
         private static void ReimportCreatorPrefabs()
         {
             string filter = "t:Prefab";
-            string[] prefabsGUIDs = AssetDatabase.FindAssets(filter, new[] {"Assets/Innoactive"});
+            string[] prefabsGUIDs = AssetDatabase.FindAssets(filter, new[] { EditorUtils.GetCoreFolder() });
```

Comment:
Is Core Folder "Assets/Innoactive" or "Assets/Innoactive/Core"?

Reply:
It actually is `Assets\Innoactive\Creator\Core` :scream:
No prefab will be updated as this is right now, thanks for pointing it out!